### PR TITLE
Use local dev lib config

### PR DESCRIPTION
**Pull request by camden11**
## Description and Context
This switches the CLI from using `@hubspot/cli-lib` for `config` to using `@hubspot/local-dev-lib`. The config module is functionally identical between both, so everything should behave exactly the same. Once this PR is merged, we'll be able to start porting everything else over to `local-dev-lib

## TODO
* This PR will need to be merged and released in tandem with this `cli-lib` PR https://github.com/HubSpot/cli-lib/pull/22
As well as a PR for the VSCode extension
* It would probably be good to stress test this pretty thoroughly before doing a full release.

## Who to Notify
@brandenrodgers

### CODE REVIEW

File: packages/cli/commands/project/deploy.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const { projectNamePrompt } = require('../../lib/prompts/projectNamePrompt');
 const { buildIdPrompt } = require('../../lib/prompts/buildIdPrompt');
 const { i18n } = require('../../lib/lang');
 const { uiBetaTag } = require('../../lib/ui');
-const { getAccountConfig } = require('@hubspot/cli-lib');
+const { getAccountConfig } = require('@hubspot/cli-lib/');
```
Comment (brandenrodgers) [inline packages/cli/commands/project/deploy.js:21]: Why does this need the trailing slash?

**Comment by brandenrodgers** [comment]
The code in here lgtm (looks like tests are failing though). I'll hammer on this locally for a bit before approving though

File: packages/cli/commands/project/deploy.js (hunk 1 above)
Comment (camden11) [reply packages/cli/commands/project/deploy.js:21]: I'm not sure how that got there. This should actually be a local-dev-lib import anyway

File: packages/cli/commands/project/deploy.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const { projectNamePrompt } = require('../../lib/prompts/projectNamePrompt');
 const { buildIdPrompt } = require('../../lib/prompts/buildIdPrompt');
 const { i18n } = require('../../lib/lang');
 const { uiBetaTag } = require('../../lib/ui');
-const { getAccountConfig } = require('@hubspot/cli-lib');
+const { getAccountConfig } = require('@hubspot/local-dev-lib');
```
Comment (brandenrodgers) [inline packages/cli/commands/project/deploy.js:21]: Missing `/config` here I think

**Comment by brandenrodgers** [comment]
I cross checked all of the exports from `local-dev-lib/config`. There are a few more exports that you may want to update so they pull from local-dev-lib:
- getAccountId (exported from the old `commonOpts` file in cli-lib)
  - Edit: I realized this is pulling from `cli/lib/commonOpts` 😨. We should probably look into fixing up that file to remove confusion between those utils and the ones exported from local-dev-lib
- isConfigFlagEnabled (I think I saw one or two missing updates for this one)
- getEnv (One or two missing updates here too)
-

**Review by brandenrodgers — APPROVED** [review]
Tested locally and this lgtm! We should make sure there isn't any other WIP PR's that need to be merged soon because this should probably sit on the beta release for a bit so we can be sure that there aren't any bugs

**Pull request merged**